### PR TITLE
fix: change convert.exe syntax to avoid clash in Windows

### PR DESCRIPTION
**Pull request by rodolforg**
On Windows (I built with CMake) for some weird reason calling ImageMagick's convert calls Windows/System32/convert.exe . It's odd because my Path env var seems to list the mingw64 directory first, but a solution is using the 'magick' syntax, where you just call 'magick' to convert as per https://imagemagick.org/script/command-line-tools.php

The problem is widely known as per https://stackoverflow.com/questions/3060205/error-invalid-parameter-fom-imagemagick-convert-on-windows , I was getting the same 'Invalid Parameter' error

The patch uses 'magick convert' (legacy syntax), instead of 'magick' for compatibility with ImageMagick 6, since many distros don't have official packages for ImageMagick 7, despite the release being over 5 years old:

    https://packages.debian.org/search?searchon=sourcenames&keywords=imagemagick
    https://packages.fedoraproject.org/pkgs/ImageMagick/ImageMagick/
    https://packages.ubuntu.com/search?keywords=imagemagick

The reason is that v7's syntax is not fully compatible with v6 but it our syntax seems safe

TL;DR:
Instead of using ImageMagick with 'convert' use 'magick convert' since Win has another 'convert' program

Supersedes: #2951

### CODE REVIEW

**Review by ice0 — APPROVED** [review]

**Comment by ice0** [comment]
Merged. Thank you!
